### PR TITLE
Add union-find query module docs

### PR DESCRIPTION
**Pull request by antepusic**
Added documentation for the union-find MAGE query module (https://github.com/memgraph/mage/pull/53).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,14625 @@
...
+        "clsx": "^1.1.1",
+        "disqus-react": "^1.0.11",
+        "react": "^16.8.4",
+        "react-dom": "^16.8.4",
+        "react-facebook-pixel": "^1.0.4",
+        "react-hotjar": "^2.2.1"
+      }
+    },
+    "node_modules/@algolia/autocomplete-core": {
+      "version": "1.2.1",
```
Comment (jmatak) [inline package-lock.json:23]: Do not include this file in commit

**Review by jmatak — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (antepusic) [reply package-lock.json]: Fixed!

**Review by jmatak — APPROVED** [review]

File: mage/query-modules/python/union-find.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+---
+id: union-find
+title: union_find
+sidebar_label: union_find
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
```
Comment (jmatak) [inline mage/query-modules/python/union-find.md:8]: Before merging, please update `mage/overview.md` file as you did in README of implementation :smile:

**Review by jmatak — CHANGES_REQUESTED** [review]

**Review by g-despot — APPROVED** [review]

File: mage/query-modules/python/union-find.md (hunk 2 above)
Comment (antepusic) [reply mage/query-modules/python/union-find.md:8]: Not sure if I understand this entirely :slightly_smiling_face: 
If you take a look at the `mage/overview.md` file under Files changed, you can see that I made some changes to it.

Comment (jmatak) [reply mage/query-modules/python/union-find.md:8]: My bad, was looking in outdated version of this PR :smile:

File: mage/query-modules/python/union-find.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+* `nodes1: Union[Vertex, List[Vertex]]` ➡ First value (or list thereof) in connectedness calculation.
+* `nodes2: Union[Vertex, List[Vertex]]` ➡ Second value (or list thereof) in connectedness calculation.
+* `mode: str("pairwise")` ➡ Mode of combining `nodes1` and `nodes2`. Can be ***p*** or ***pairwise*** for a pairwise product, or ***c*** or ***cartesian*** for a Cartesian product of the arguments. Pairwise by default.
+* `update: bool(True)` ➡ Specifies whether the disjoint-set data structure should be reinitialized. Enabled by default. If the graph has been modified since the previous call of this procedure, turning `update` off ensures that the changes are not visible in the output.
+
+#### Output:
+
+* `node1` ➡ Node in `nodes1`.
+* `node2` ➡ Node in `nodes2`.
+* `connected` ➡ `True` if the above nodes are in the same connected component of the graph.
```
Comment (jmatak) [inline mage/query-modules/python/union-find.md:59]: Set the output type here please

Comment (antepusic) [reply mage/query-modules/python/union-find.md]: Done!

**Pull request merged**